Extensions/ParticleSystem/particleemitterobject.js

```diff
@@ -1,56 +1,169 @@
 /**
-
-GDevelop - Particle System Extension
-Copyright (c) 2010-2016 Florian Rival (Florian.Rival@gmail.com)
-This project is released under the MIT License.
+ * @fileoverview
```

Comment:
If possible I'd like to keep the same header everywhere:

```
/*
 * GDevelop JS Platform (or GDevelop - Extension name) (or GDevelop Core) 
 * Copyright 2018-present Florian Rival (Florian.Rival@gmail.com). All rights reserved.
 * This project is released under the MIT License.
 */
```

The reason is that it's not too large (so don't distract too much) and gives the main information (the license). Adding annotations here won't be useful.

Reply:
This one was very ugly and not like the others, that is why I replaced it, but ok

Reply:
It's fine to change it to the one I put before then - what matter to me is the consistency (so that if one day we want to change, we just have to do a clever search/replace in the whole codebase).

Reply:
Ok will fix that. But not yet I am typing all the sprite extension right now (It will take some time)

Reply:
Yeah don't worry about typing all this extension - we can do later progressively.

Reply:
Can we really remove the annotations for this header? 😇 Consistency with other headers is more important :) 

```
/*
 * GDevelop - Particle System Extension
 * Copyright 2008-present Florian Rival (Florian.Rival@gmail.com). All rights reserved.
 * This project is released under the MIT License.
 */
```

Reply:
ok restored the header